site/en/guide/estimators.md

```diff
@@ -112,8 +113,8 @@ of the following four steps:
 4.  **Call a training, evaluation, or inference method.**
     For example, all Estimators provide a `train` method, which trains a model.
 
-        # my_training_set is the function created in Step 1
-        estimator.train(input_fn=my_training_set, steps=2000)
+        # `data_input` is the function created in Step 1
+        estimator.train(data_input=my_training_set, steps=2000)
```

Comment:
`input_fn` is the param name for `.train`: https://github.com/tensorflow/tensorflow/blob/r1.12/tensorflow/python/estimator/estimator.py#L280